docs/node-mixin/dashboards/use.libsonnet

```diff
@@ -0,0 +1,200 @@
+local g = import 'grafana-builder/grafana.libsonnet';
+
+{
+  grafanaDashboards+:: {
+    'node-cluster-rsrc-use.json':
+      local legendLink = '%s/dashboard/file/k8s-node-rsrc-use.json' % $._config.grafana_prefix;
+
+      g.dashboard('USE Method / Cluster')
+      .addRow(
+        g.row('CPU')
+        .addPanel(
+          g.panel('CPU Utilisation') +
+          g.queryPanel(|||
+            (
+              instance:node_cpu_utilisation:avg_rate1m
+            *
+              instance:node_num_cpu:sum
```

Comment:
This is more complicated than it needs to be, why are you multiplying by the number of cpus and then undoing that by dividing?

Reply:
The original author of this impression is multiplying by the number of CPUs of an individual instance and then divides by the _total_ number of CPUs to normalize the CPU utilization of a single instance by the fraction of its own CPUs of the total number of CPUs. In that way, if stacked (like in this panel), the stacked result will represent the total utilization.

Reply:
What _could_ make things simpler is if we introduced a rule `instance_cpu:node_cpu_utilisation:rate1m` (now really without the `avg`). Then we would save the multiplication. But we need the average rule for the per-instance dashboard.

Reply:
But then you would still need to sum up the individual CPU utilizations here... I guess it's not really simpler either way. I'd just leave it as it for now.

Reply:
I don't think most users would care about per-cpu utilisation, just overall machine cpu. It only really comes up if you're pinning processes to cores.